usaspending_api/awards/v2/filters/view_selector.py

```diff
@@ -232,6 +241,10 @@ def can_use_view(filters, view_name):
             not compatible with the materialized view.
         '''
         if rules['type'] == 'list':
+            if rules.get("is_present") and key in filters:
+                for field in filters[key]:
+                    if field.get(rules.get("key")):
+                        return False
```

Comment:
Why to we need both this and the try/catch? I'm not clear what this is trying to accomplish, but I sense a code smell.

Reply:
I don't like this snippet myself, but it works and is a distinctly separate check from the try-catch test. I am open to recommendations.